packages/react/src/index.ts

```diff
@@ -139,6 +139,9 @@ export { default as PopupContent } from './components/Popup/PopupContent'
 export * from './components/Portal/Portal'
 export { default as Portal } from './components/Portal/Portal'
 
+export * from './components/Portal/PortalInner'
+export { default as PortalInner } from './components/Portal/PortalInner'
```

Comment:
I think it would be easier to just use `createPortal` in the prototype. We would need to add more documentation if we start exposing `PortalInner`

Reply:
Replaced